sdks/python/snaptrade_client/model/account_holdings.pyi

```diff
@@ -54,8 +54,8 @@ class AccountHoldings(
                 class MetaOapg:
                     
                     @staticmethod
-                    def items() -> typing.Type['Balance']:
-                        return Balance
+                    def items() -> typing.Type['BalanceSchema']:
+                        return BalanceSchema
```

Comment:
@dphuang2 I don't think this is right. `BalanceSchema` isnt defined in this file, and `Balance` correctly refers to the model

Reply:
I think we only want to do this if the model was imported as `{typename}Schema`. Why is it for some but not for others?

Reply:
Is it causing an error?

Reply:
![image](https://github.com/passiv/snaptrade-sdks/assets/35413716/e327d1e5-e129-42fe-9144-a9de4abf91b0)

yeah BalanceSchema is not defined so this will definitely cause issues

Reply:
so we could just have 2 different list partials. or we could have some condition in the list partial

Reply:
hm makes sense

Reply:
now fixed 🎉